infrastructure-playbooks/rolling_update.yml

```diff
@@ -363,20 +363,25 @@
       command: "{{ docker_exec_cmd_update_osd|default('') }} ceph --cluster {{ cluster }} versions"
       register: ceph_versions
       delegate_to: "{{ groups[mon_group_name][0] }}"
+      when:
+        - not jewel_minor_update
 
     - name: set_fact ceph_versions_osd
       set_fact:
         ceph_versions_osd: "{{ (ceph_versions.stdout|from_json).osd }}"
       delegate_to: "{{ groups[mon_group_name][0] }}"
+      when:
+        - not jewel_minor_update
 
     # length == 1 means there is a single osds versions entry
     # thus all the osds are running the same version
     - name: complete osds upgrade
       command: "{{ docker_exec_cmd_update_osd|default('') }} ceph --cluster {{ cluster }} osd require-osd-release luminous"
       delegate_to: "{{ groups[mon_group_name][0] }}"
       when:
-        - (ceph_versions.stdout|from_json).osd | length == 1
+        - (ceph_versions.get('stdout', '{}')|from_json).get('osd', {}) | length == 1
```

Comment:
You need to add quotes to the ``{}`` in ``.get('osd', {})`` like you've done with the first one.

Reply:
nevermind, quotes on the first were needed because of ``from_json``, the next one does not need it.